rskj-core/src/jmh/java/co/rsk/jmh/web3/plan/EthCallPlan.java

```diff
@@ -0,0 +1,98 @@
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package co.rsk.jmh.web3.plan;
+
+import co.rsk.jmh.helpers.BenchmarkHelper;
+import co.rsk.jmh.web3.BenchmarkWeb3Exception;
+import co.rsk.jmh.web3.e2e.RskModuleWeb3j;
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.TearDown;
+import org.openjdk.jmh.infra.BenchmarkParams;
+import org.web3j.protocol.core.methods.request.Transaction;
+import org.web3j.protocol.core.methods.response.EthTransaction;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.util.Iterator;
+
+@State(Scope.Benchmark)
+public class EthCallPlan extends BasePlan {
+
+    private Iterator<Transaction> transactionsContractCall;
+    private org.web3j.protocol.core.methods.response.Transaction contractCallTransaction;
+    private BigInteger blockNumber;
+    private RskModuleWeb3j.EthCallArguments ethCallArguments;
+
+    @Override
+    @Setup(Level.Trial) // move to "Level.Iteration" in case we set a batch size at some point
+    public void setUp(BenchmarkParams params) throws BenchmarkWeb3Exception {
+        super.setUp(params);
+
+        blockNumber = web3Connector.ethBlockNumber();
+        ethCallArguments = buildEthCallArguments();
+    }
+
+    private org.web3j.protocol.core.methods.response.Transaction setupContractCallTransaction() throws IOException {
+        EthTransaction transactionResponse = rskModuleWeb3j.ethGetTransactionByHash(configuration.getString("eth.transactionContractCallHash")).send();
+        return transactionResponse.getResult();
+    }
+
+    @TearDown(Level.Trial) // move to "Level.Iteration" in case we set a batch size at some point
+    public void tearDown() throws InterruptedException {
+        // wait for new blocks to have free account slots for transaction creation
+        BenchmarkHelper.waitForBlocks(configuration);
+    }
+
+    public org.web3j.protocol.core.methods.response.Transaction getContractCallTransaction() {
+        try {
+            contractCallTransaction = setupContractCallTransaction();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        return contractCallTransaction;
+    }
+
+    private RskModuleWeb3j.EthCallArguments buildEthCallArguments() {
+        RskModuleWeb3j.EthCallArguments args = new RskModuleWeb3j.EthCallArguments();
+        org.web3j.protocol.core.methods.response.Transaction contractCallTransaction = getContractCallTransaction();
```

Comment:
As discussed internally, a contract should be pre-created during setup and then `eth_call` should call that contract. Here you are using the data of a Remasc Transaction (as per configuration) to fill `args`.

Basically, `args` should represent a call to the pre-created contract.